src/very_basic_operations.py

```diff
@@ -2,5 +2,17 @@ def add(a, b):
     """Basic function which adds two numbers"""
     return a + b 
 
+def factorial(number):
+    n = 1
+    if type(number) != int or number < 0:
```

Comment:
In `if type(number) != int` you check that `number` is not instance of `int`. Better way to do this is `if not isinstance(number, int)`.